cortex-mixin/dashboards/alertmanager.libsonnet

```
@@ -83,5 +87,136 @@ local utils = import 'mixin-utils/utils.libsonnet';
         $.panel('Latency') +
         utils.latencyRecordingRulePanel('cortex_request_duration_seconds', $.jobSelector($._config.job_names.gateway) + [utils.selector.re('route', 'api_v1_alerts|alertmanager')])
       )
+    )
+    .addRow(
+      $.row('Replication')
+      .addPanel(
+        $.panel('Per %s Tenants' % $._config.per_instance_label) +
+        $.queryPanel(
+          'max by(%s) (cortex_alertmanager_tenants_owned{%s})' % [$._config.per_instance_label, $.jobMatcher('alertmanager')],
+          '{{%s}}' % $._config.per_instance_label
+        ) +
+        $.stack
+      )
+      .addPanel(
+        $.panel('Per %s Alerts' % $._config.per_instance_label) +
+        $.queryPanel(
+          'sum by(%s) (cortex_alertmanager_alerts{%s})' % [$._config.per_instance_label, $.jobMatcher('alertmanager')],
+          '{{%s}}' % $._config.per_instance_label
+        ) +
+        $.stack
+      )
+      .addPanel(
+        $.panel('Per %s Silences' % $._config.per_instance_label) +
+        $.queryPanel(
+          'sum by(%s) (cortex_alertmanager_silences{%s})' % [$._config.per_instance_label, $.jobMatcher('alertmanager')],
+          '{{%s}}' % $._config.per_instance_label
+        ) +
+        $.stack
+      )
+    )
+    .addRow(
+      $.row('Tenant Configuration Sync')
+      .addPanel(
+        $.panel('Syncs/sec') +
+        $.queryPanel(
+          [
+            |||
+              sum(rate(cortex_alertmanager_sync_configs_total{%s}[$__rate_interval]))
+              -
+              sum(rate(cortex_alertmanager_sync_configs_failed_total{%s}[$__rate_interval]))
+            ||| % [$.jobMatcher('alertmanager'), $.jobMatcher('alertmanager')],
+            'sum(rate(cortex_alertmanager_sync_configs_failed_total{%s}[$__rate_interval]))' % $.jobMatcher('alertmanager'),
+          ],
+          ['success', 'failed']
+        )
+      )
+      .addPanel(
+        $.panel('Syncs/sec (By Reason)') +
+        $.queryPanel(
+          'sum by(reason) (rate(cortex_alertmanager_sync_configs_total{%s}[$__rate_interval]))' % $.jobMatcher('alertmanager'),
+          '{{reason}}'
+        )
+      )
+      .addPanel(
+        $.panel('Ring Check Errors/sec') +
+        $.queryPanel(
+          'sum (rate(cortex_alertmanager_ring_check_errors_total{%s}[$__rate_interval]))' % $.jobMatcher('alertmanager'),
+          'errors'
+        )
+      )
+    )
+    .addRow(
+      $.row('Sharding Runtime State Sync')
+      .addPanel(
+        $.panel('Syncs/sec') +
+        $.queryPanel(
+          'sum by(outcome) (rate(cortex_alertmanager_state_initial_sync_completed_total{%s}[$__rate_interval]))' % $.jobMatcher('alertmanager'),
+          '{{outcome}}'
+        )
+      )
+      .addPanel(
+        $.panel('Sync duration') +
```

[nit] Just to be super clear

```suggestion
        $.panel('Initial sync duration') +
```